server/venueless/core/models/chat.py

```diff
@@ -65,9 +69,24 @@ def serialize_public(self):
             "content": self.content,
             "edited": self.edited.isoformat() if self.edited else None,
             "replaces": self.replaces_id,
+            "reactions": reactions,
         }
 
 
+class ChatEventReaction(models.Model):
+    reaction = models.CharField(max_length=4)
```

Comment:
Is that 4 bytes or 4 unicode glyphs? If it's 4 bytes, it prevents us from doing e.g. skin colors or rainbow flag.

If it's 4 unicode glyphs, it's 4 times as many as we need *and* prevents a future with custom emoji. And who doesn't want a future with custom emoji?

Proposal: Just switch to TextField. We're PostgreSQL-only, and there's virtually no performance or storage space benefit of varchar over text on PostgreSQL (there is on other databases).